ios/Sources/Component/LoadingStatus.swift

```diff
@@ -0,0 +1,6 @@
+public enum LoadingStatus<T> {
```

Comment:
Looks like `loaded` could remind me of meaning of error as well.
In my opinion, `pending`, `loading(or running)`, `success`, `failure` might be better for names. 
ref: https://github.com/ra1028/SwiftUI-Hooks/blob/main/Sources/Hooks/AsyncStatus.swift

Reply:
@ry-itto 
How about this one?

Reply:
This `LoadingStatus` is equal to KMM module's `LoadingState`. So I think it shouldn't change to other name cases.

If we don't need to use KMM module's `LoadingState` in Model module scope, I think your opinion will be better.